src/main/java/frc/robot/util/PathPlannerCommands.java

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.util;
+
+import com.pathplanner.lib.commands.FollowPathHolonomic;
+import com.pathplanner.lib.path.PathPlannerPath;
+import com.pathplanner.lib.util.HolonomicPathFollowerConfig;
+import com.pathplanner.lib.util.ReplanningConfig;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.DriverStation.Alliance;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.subsystem.DriveSubsystem;
+import frc.robot.subsystem.PositioningSubsystem;
+
+/** Add your docs here. */
+public class PathPlannerCommands {
+    private static final ReplanningConfig REPLANNING_CONFIG = new ReplanningConfig();
```

Comment:
Dynamic replanning seems to be a new feature this year. I think we should turn it on, but I'm open to discussion.

Either way, we explicitly turn it on or off when we instantiate the `ReplanningConfig`. Whether or not we have dynamic replanning enabled should be obvious at a glance without needing to refer to the documentation or library source code to look up the defaults.